src/libstd/sys/unix/os.rs

```diff
@@ -30,7 +30,7 @@ use sys::c;
 use sys::fd;
 use vec;
 
-const GETCWD_BUF_BYTES: usize = 2048;
+const GETCWD_BUF_BYTES: usize = 512;
```

Comment:
It's ok to just go ahead and delete this constant and inline its value, I'm not sure that it's necessarily buying us a whole lot up here